src/main/webapp/js/adminHome.js

```diff
@@ -0,0 +1,119 @@
+            $("#addInstructorResultPanel div.panel-heading").html(panelHeader);
+            if (paramsCounter < paramsList.length) {
+                addInstructorByAjaxRecursively();
+            } else {
+                enableAddInstructorForm();
+            }
+        },
+        success : function(data) {
+            var rowText = createRowForResultTable(data.instructorShortName, data.instructorName, 
+                                                  data.instructorEmail,data.instructorInstitution,
+                                                  data.instructorAddingResultForAjax, data.statusForAjax);
+            $("#addInstructorResultTable tbody").append(rowText);
+            if ((!data.instructorAddingResultForAjax) && (isInputFromFirstPanel)) {
+                var errorInstructorsText = $("#addInstructorDetailsSingleLine").val() + instructorDetailsList[paramsCounter] + "\n";
+                $("#addInstructorDetailsSingleLine").val(errorInstructorsText);
+            }
+            paramsCounter++;
+            var panelHeader = "<strong>Result (" + paramsCounter + "/" + paramsList.length + ")</strong>";
+            $("#addInstructorResultPanel div.panel-heading").html(panelHeader);
+            if (paramsCounter < paramsList.length) {
+                addInstructorByAjaxRecursively();
+            } else {
+                enableAddInstructorForm();
+            }
+        }
+    });
+}
+
+function addInstructorByAjax() {
+    $("#addInstructorResultPanel").show();    // show the hidden panel
+    
+    var multipleLineText = $("#addInstructorDetailsSingleLine").val();    // get input from the first panel
+    multipleLineText = multipleLineText.trim();
+    
+    if (multipleLineText.length == 0) {
+        var instructorDetails = $("#instructorName").val() + "|" + $("#instructorEmail").val() + "|" + $("#instructorInstitution").val();
+        instructorDetailsList = [instructorDetails];
+        var params = "instructorshortname=" + $("#instructorShortName").val() +
+                     "&instructorname=" + $("#instructorName").val() +
+                     "&instructoremail=" + $("#instructorEmail").val() +
+                     "&instructorinstitution=" + $("#instructorInstitution").val();
+        paramsList = [params];
+        isInputFromFirstPanel = false;
+    } else {
+        instructorDetailsList = multipleLineText.split("\n");
+        paramsList = [];
+        for(var i = 0; i < instructorDetailsList.length; i++) {
+            instructorDetailsList[i] = instructorDetailsList[i].replace(/\t/g,"|");
+            paramsList[i] = "instructordetailssingleline=" + instructorDetailsList[i];
+        }
+        isInputFromFirstPanel = true;
+    }
+    paramsCounter = 0;
+    $("#addInstructorResultTable tbody").html("");    // clear table
+    $("#addInstructorDetailsSingleLine").val("");    // clear input form
```

Comment:
I think it doesn't matter since if isInputFromFirstPanel is `false`, the form is empty anyway.